packages/oauth/oauth-client-browser/example/src/main.tsx

```diff
@@ -10,11 +10,12 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
     <AuthProvider
       // dev-env
-      // plcDirectoryUrl="http://localhost:2582"
-      // handleResolver="http://localhost:2584"
+      plcDirectoryUrl="http://localhost:2582"
+      handleResolver="http://localhost:2584"
       // production
-      plcDirectoryUrl={undefined}
-      handleResolver="https://bsky.app"
+      // plcDirectoryUrl={undefined}
+      // handleResolver="https://bsky.social"
```

Comment:
Just double checking these changes are intentional, not from development/debugging.

Reply:
I use this a lot in dev, which means that I often have to change those params. I figured since this is not the right place for an example app anyways (docs site probably is), I could repurpose this for dev.